aws-s3-controller.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: aws-s3-controller
+  version: 1.0.21
+  epoch: 0
+  description: ACK service controller for Amazon Simple Storage Service (S3)
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 2be16650fa025711d45f3f8042904e6ff0c211fd
+      repository: https://github.com/aws-controllers-k8s/s3-controller.git
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X main.version=v${{package.version}}
+        -X main.buildHash=$(git rev-parse HEAD)
+        -X main.buildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+      packages: ./cmd/controller
+      output: controller
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "Compatibility package to place binaries in the location expected by aws-s3-controller"
+    dependencies:
+      runtime:
+        - ${{package.name}}
```

Comment:
we can omit this given for fips builds we can use the same compat in the future. 💭